### ISSUE TITLE
[Bug]: Referral system menu is displayed on the main page after a fresh installation

### ISSUE THREAD

**Issue by MrWeez** [Ctrlpanel-gg/panel#1023]
### What is your request about?

With a fresh installation, the Referral system is disabled by default in the settings, but at the same time there is a menu with information about the referral program on the home page.

![Screenshot from 2024-11-12 00-59-30](https://github.com/user-attachments/assets/2efea236-ef62-4f0e-9676-5986b4696839)
![Screenshot from 2024-11-12 01-00-14](https://github.com/user-attachments/assets/a65c6aee-8437-46b8-88a5-8c8ebd2fabf4)

### Branch

development

**Comment by 1day2die** [comment]
![grafik](https://github.com/user-attachments/assets/4e64e478-d4d7-41a2-aa5c-b7bbcc973c9f)
![grafik](https://github.com/user-attachments/assets/e6258279-167e-49dd-b64a-7ebe0e6e6e8f)


cannot confirm

**Comment by 1day2die** [comment]
does is go away if you save the referral settigns?

**Comment by MrWeez** [comment]
> does is go away if you save the referral settigns?

Yep

**Comment by MrWeez** [comment]
I will try to install it again to double-check this bug.

**Comment by 1day2die** [comment]
i already know the cause

**Comment by MrWeez** [comment]
> cannot confirm on a fresh install with a fresh DB
> 
> ![grafik](https://private-user-images.githubusercontent.com/8725848/385241858-fa23da17-9c90-4ebf-84ce-d7928393196d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE0MTk4MzksIm5iZiI6MTczMTQxOTUzOSwicGF0aCI6Ii84NzI1ODQ4LzM4NTI0MTg1OC1mYTIzZGExNy05YzkwLTRlYmYtODRjZS1kNzkyODM5MzE5NmQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTExMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMTJUMTM1MjE5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzVkNTk4NDg3NGI1YjNkYmRjOTdiMzUzMWM0NWY3OTI1ZGVhNWYxZTQ5NWRkNjQ0Mzg2ZWY2NDY4YjgzOGRmOSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ._sXrT2ZabKvYBIiSlVBvkxf7JVzHSySDfzGieV4QeVg)

Okay, apparently it was some kind of cache or something. I have now installed everything in a new, fresh database, but I still have the old names on the home page, although everything is fine in the settings

**Comment by 1day2die** [comment]
the old names?

**Comment by MrWeez** [comment]
> the old names?

Yes, I have no idea why, apparently the browser cache, because in the database the name of the credits is "Credits", although for some reason on the main page it is "Testics" that have been since the time when I checked the settings, while I just reinstalled CtrlPanel via rm -rf /var/www/controlpanel

**Comment by MrWeez** [comment]
> while I just reinstalled CtrlPanel via rm -rf /var/www/controlpanel
So it's definitely not a bug
